changelog_unreleased/cli/11369.md

```diff
@@ -0,0 +1,7 @@
+#### Updated failure message to be more informative (#11369 by @webark)
+
+Updated the "Forgot to run Prettier?" to "Run Prettier to fix."
+
+This keeps the same spirit of the message, but is less likely to be
+misinterpreted as it's a more formal message rather than being
+somewhat familial.
```

Comment:
I forgot how we wrap this normally, should the on one line?

Reply:
If we wanted to enforce wrapping, we'd have `proseWrap: always` in `.prettierrc`.